test/controllers/users.test.js

```diff
@@ -0,0 +1,82 @@
+const supertest = require('supertest');
+const app = require('../../app');
+const models = require('../../app/models/index');
+
+const request = supertest(app);
+
+const bodyUser = (firstName, lastName, email, password) => ({
+  first_name: firstName,
+  last_name: lastName,
+  email,
+  password
+});
+
+describe('UserController', () => {
```

Comment:
We normally name like this
```suggestion
describe('user controller', () => {
```